### ISSUE TITLE
Would it be possible to add address book functions?

### ISSUE THREAD

**Issue by ThrownLemon** [dl3br/rx-monero-wallet#3]
Firstly, love your work, i'm using your package in my current project and it's made life a lot easier!

Secondly, would it possible for you to add the following functions?

get_address_book
add_address_book
delete_address_book

and if possible:

create_wallet
open_wallet

and last but not least can you add some examples for some of the other functions not shown in the readme.

Cheers,
Travis

**Comment by dl3br** [comment]
i did quickly without testing

could u please test it and then i will publish it to npm

cheers
d

**Comment by ThrownLemon** [comment]
had some trouble using the latest in my project how would i import the latest?

In package.json i had linked the git repo like this under dependencies but it gave errors saying it wasn't packaged correctly.

"rx-monero-wallet": "git://github.com/dl3br/rx-monero-wallet.git"

**Comment by dl3br** [comment]
```
npm i  https://github.com/dl3br/rx-monero-wallet/tarball/master --save
```

**Comment by ThrownLemon** [comment]
Found the issue, looks like the offending line is 9 of lib\request.ts
`import 'rxjs/add/observable/scan';`

You need to replace it with the following:
`import { scan } from 'rxjs/operators';`

**Comment by ThrownLemon** [comment]
Tested the new functions and everything seems to be working after the above change. 

It should be okay to publish this change.

Thanks again!

**Comment by dl3br** [comment]
removed scan operator, and published to npm, could u check if its working correctly?

**Comment by ThrownLemon** [comment]
Something weird...

`npm i rx-monero-wallet --save`

```
+ rx-monero-wallet@1.0.16
added 1 package from 1 contributor in 70.258s
```

`npm start`

```
ERROR in node_modules/rx-monero-wallet/lib/wallet.ts(89,28): error TS2554: Expected 0 arguments, but got 1.
node_modules/rx-monero-wallet/lib/wallet.ts(89,44): error TS2339: Property 'times' does not exist on type 'Xmr'.
node_modules/rx-monero-wallet/lib/wallet.ts(94,22): error TS2554: Expected 0 arguments, but got 1.
node_modules/rx-monero-wallet/lib/wallet.ts(94,35): error TS2339: Property 'div' does not exist on type 'Atomic'.
```

It seems like your dependencies didn't install for me?

This is the entry in package-lock,json

```
"rx-monero-wallet": {
      "version": "1.0.16",
      "resolved": "https://registry.npmjs.org/rx-monero-wallet/-/rx-monero-wallet-1.0.16.tgz",
      "integrity": "sha512-ylmDKtD2AvrPm3bJ3QLTVD2+gicZYXVHxI9i4HpC9vYWhY9pI838sKWGbykcMym7LcR8ZETznFfTvOBz4VLtvQ==",
      "requires": {
        "big.js": "3.2.0",
        "random-js": "1.0.8",
        "rxjs-fetch": "2.1.6"
      }
    },
```

**Comment by dl3br** [comment]
i moved the @types back to dependencies instead of devVDependencies. i published to npm rx-monero-wallet@1.0.17

can u test again. hopefully it was that, but it doesnt make that much sense

**Comment by ThrownLemon** [comment]
Yep, that seems to have fixed it for me and the new functions are also working.

Thanks!
